price-estimator/src/filter.rs

```diff
@@ -68,6 +73,30 @@ async fn handle_rejection(err: Rejection) -> Result<impl Reply, Infallible> {
     Ok(warp::reply::with_status(json, code))
 }
 
+/// Validate a request of the form
+/// `/minimum-order-size-owl` and answer it.
+fn minimum_order_size_owl(
+    economic_viability: Arc<dyn EconomicViabilityComputing>,
+) -> impl Filter<Extract = (Json,), Error = Rejection> + Clone {
+    warp::path!("minimum-order-size-owl")
```

Comment:
Maybe this route could be `minimum-order-size/:token-id` and then reject if the token ID is not `0`? That way we just have this one route for minimum trades, and we can just add support for other token IDs in the future.

Reply:
I have a hunch that it could be a good idea to really keep the routes separate to make it easier to move the more advanced route in the future without breaking compatibility. This way we don't need to spend thinking about the best api design for the route with tokens now.

Reply:
Sounds good to me!